subsys/bluetooth/host/hci_core.h

```diff
@@ -64,6 +65,36 @@ enum {
 #define BT_DEV_PERSISTENT_FLAGS (BIT(BT_DEV_ENABLE) | \
 				 BIT(BT_DEV_PRESET_ID))
 
+enum {
+	BT_ADV_CREATED,
+
+	BT_ADV_RPA_VALID,
+
+	BT_ADV_LIMITED,
+
+	BT_ADV_NUM_FLAGS,
+};
+
+struct bt_le_ext_adv {
+	/* ID Address used for advertising */
+	u8_t                    id;
+
+	/* Advertising handle */
+	u16_t			handle;
+
+	/* Current local Random Address */
+	bt_addr_le_t		random_addr;
+
+	ATOMIC_DEFINE(flags, BT_ADV_NUM_FLAGS);
+
+#if defined(CONFIG_BT_ADV_EXT)
+	const struct bt_le_ext_adv_cb *cb;
```

Comment:
Would it make sense to try to make Kconfig and C-APIs with consistent name spaces? You now have BT_ADV_EXT in one and bt_le_ext_adv in the other.

Reply:
We don't have _LE_ in Kconfig options. But using EXT_ADV consistently is a good idea.
Although this should be aligned with CTLR_ADV_EXT as well, but I don't wish to include that change into this PR.
@cvinayak Maybe you can update in your branch(es) if you agree?